pubtools/_quay/clear_repo.py

```diff
@@ -0,0 +1,256 @@
+import logging
+
+from .signature_remover import SignatureRemover
+from .quay_api_client import QuayApiClient
+from .untag_images import untag_images
+from .utils.misc import (
+    setup_arg_parser,
+    add_args_env_variables,
+    send_umb_message,
+    get_internal_container_repo_name,
+)
+
+LOG = logging.getLogger()
+logging.basicConfig()
+LOG.setLevel(logging.INFO)
+
+CLEAR_REPO_ARGS = {
+    ("--repository",): {
+        "help": "External repository to clear. Must be in format <namespace>/<repo>.",
+        "required": True,
+        "type": str,
+    },
+    ("--namespace",): {
```

Comment:
can we rename it to org or quay-org? namespace could be misleading because it's already part of repo

Reply:
Changed. I assumed you also wanted it changed for the remove-repo entrypoint